Gopkg.toml

```diff
@@ -96,15 +96,15 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.10.0"
+  version = "kubernetes-1.13.2"
```

Comment:
We have currently put the k8s version updates on hold. @jackfrancis can provide details about this.